### DISCUSSION TITLE
How to work with adresses?

### DISCUSSION THREAD

**Discussion by java-juggled-jazz** [osmdroid/osmdroid#1903] [Q&A]
Hello there! How can I get address of point or set point by address? Now I need to work with one city. Should I use libs or packages? Where can I get it?

**Comment by MKergall** [comment]
You can use OSMBonusPack lib. 
See: https://github.com/MKergall/osmbonuspack/wiki/Features#geocoding-and-reverse-geocoding

**Reply by java-juggled-jazz** [reply]
Thanks.
